app/src/androidTest/java/com/triviagenai/options/FakeUserPreferenceRepository.kt

```diff
@@ -0,0 +1,15 @@
+package com.triviagenai.options
+
+import com.triviagenai.triviagen.options.domain.repository.UserPreferenceRepository
+
+class FakeUserPreferenceRepository(): UserPreferenceRepository {
```

Comment:
I don't think this class is being used any more.

Reply:
It is used in NavigationTest, so I can delete this when resolving conflict with newest NavigationTest